### ISSUE TITLE
Life Regen not working when [Food] enabled

### ISSUE THREAD

**Issue by nxPublic** [nxPublic/ValheimPlus#21]
I am aware of this issue and will upload a 0.6A version by tomorrow to fix this issue.

**Comment by nxPublic** [comment]
You can disable food and the multiplier and it will work again.

```ini
[Food]
enabled=false
; enable/disable Food changes

foodDuration=0
; default is 0, this is a percent value. (default + foodDuration%)
; 100 is 100% increased food duration.
; currently does not properly show in item tooltips

```

**Comment by nxPublic** [comment]
This issue will be fixed in 0.7

**Comment by nxPublic** [comment]
Fixed in 0.7
